src/input/keyconfig.cpp

```diff
@@ -173,13 +173,15 @@ static void initializeKeyFunctions(std::vector<KeyFunctionInfo>& entries)
 	entries.emplace_back(KeyFunctionInfo(InputContext::GAMEPLAY,        KeyMappingType::ASSIGNABLE,  kf_SelectUnits(DST_UNUSED, DS_ALL_UNITS),                      "SelectAllUnits",               N_("Select EVERY unit"),                            {{ KeyMappingSlot::PRIMARY, { KEY_CODE::KEY_LCTRL,      KEY_CODE::KEY_U             } }}));
 	entries.emplace_back(KeyFunctionInfo(InputContext::GAMEPLAY,        KeyMappingType::ASSIGNABLE,  kf_SelectUnits(DST_VTOL),                                      "SelectAllVTOLs",               N_("Select all VTOLs"),                             {{ KeyMappingSlot::PRIMARY, { KEY_CODE::KEY_LCTRL,      KEY_CODE::KEY_V             } }}));
 	entries.emplace_back(KeyFunctionInfo(InputContext::GAMEPLAY,        KeyMappingType::ASSIGNABLE,  kf_SelectUnits(DST_VTOL_ARMED),                                "SelectAllArmedVTOLs",          N_("Select all fully-armed VTOLs"),                 {{ KeyMappingSlot::PRIMARY, { KEY_CODE::KEY_LSHIFT,     KEY_CODE::KEY_V             } }}));
+	entries.emplace_back(KeyFunctionInfo(InputContext::GAMEPLAY,        KeyMappingType::ASSIGNABLE,  kf_SelectNoGroupUnits(DST_VTOL_ARMED),                         "SelectAllArmedVTOLsNG",        N_("Select all fully-armed VTOLs without group"),   {{ KeyMappingSlot::PRIMARY, { KEY_CODE::KEY_LSHIFT,     KEY_CODE::KEY_V             } }}));
```

Comment:
There is a conflict between the non-group fully armed VTOL and fully armed VTOL keymapping resulting in the default binding going into the non-group one.

Simple solution is to default bind your new VTOL keybind to the left alt button.

KEY_CODE::KEY_LALT